models/comment_model.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+import (
+	"github.com/jinzhu/gorm"
+	"time"
+)
+
+type Comment struct {
+	gorm.Model
+	Id           int64      `gorm:"primary_key"`
```

Comment:
@breakingrobot Shouldn't it be an **unsigned 64 bits integer** ? Identifier and primary key **MUST NOT** be a negative integer!